.github/workflows/helm-chart-lint.yaml

```diff
@@ -41,20 +51,22 @@ jobs:
         run: |
           changed=$(ct list-changed --target-branch ${{ github.event.repository.default_branch }})
           if [[ -n "$changed" ]]; then
-            echo "CHART_CHANGED=true" >> $GITHUB_ENV
+            echo "changed=true" >> $GITHUB_OUTPUT
           fi
 
-      # run chart linting
       - name: Run chart-testing (lint)
-        run: ct lint --target-branch ${{ github.event.repository.default_branch }} --config charts/config/chart-testing-config.yaml
-
-      # Preparing a kind cluster to install and test charts on
-      - name: Create kind cluster
-        uses: helm/kind-action@v1.4.0
-        if: ${{ env.CHART_CHANGED == 'true' }}
-
-      # install the chart to the kind cluster and run helm test
-      # define charts to test with the --charts parameter
+        run: ct lint --validate-maintainers=false --target-branch ${{ github.event.repository.default_branch }}
       - name: Run chart-testing (install)
-        run: ct install --charts charts/bpdm,charts/bpdm/charts/bpdm-pool,charts/bpdm/charts/bpdm-gate,charts/bpdm/charts/bpdm-bridge-dummy,charts/bpdm/charts/bpdm-cleaning-service-dummy,charts/bpdm/charts/bpdm-orchestrator --config charts/config/chart-testing-config.yaml
```

Comment:
this is also maybe created for a reason to validate all charts in your leading repostiory